src/components/slider/slider.js

```diff
@@ -85,9 +87,10 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
     };
 
     var isDisabledParsed = attr.ngDisabled && $parse(attr.ngDisabled);
-    var isDisabledGetter = isDisabledParsed ? 
+    var isDisabledGetter = isDisabledParsed ?
       function() { return isDisabledParsed(scope.$parent); } :
       angular.noop;
+    var wrapper = angular.element(element[0].querySelector('.md-slider-wrapper'));
```

Comment:
Good question - I used it at one point while fixing this, but apparently, didn't end up needing it.  Will remove.